### ISSUE TITLE
Reminder: deprecate FacilityID AR

### ISSUE THREAD

**Issue by nathankota** [uic-utah/uic-inventory#281]
**ref:** [AR #124](https://github.com/uic-utah/uic-attribute-rules/issues/124)
And not until #259 is implemented

**Comment by nathankota** [comment]
This is on hold until data transfer is solved.

**Comment by nathankota** [comment]
no longer relevant
